### ISSUE TITLE
Document `light_*` colors?

### ISSUE THREAD

**Issue by joshorr** [borntyping/python-colorlog#124]
I was trying to figure out how to get the `bright` colors that Pycharm terminal supports.  Took my some time before I released there were undocumented colors, the `light_*` colors.

I could adjust the documentation for you if you want to added the light_* colors to the README.md, or did you want to do that, or did you want to just leave it as-is?

Figured I would open this up, as it would have saved me a bunch of time if they were in the list of colors.

**Comment by borntyping** [comment]
A PR to adjust the documentation would be appreciated!

~~I suspect the reason I never properly documented the `light_` foreground and background colours is that they aren't supported by all terminals.~~ They're also incorrectly named, and really should be called `bright`. These colours were added recently and copied the names used by the `colorama` library: https://github.com/borntyping/python-colorlog/issues/87.

**Comment by joshorr** [comment]
Yes, pycharm uses the term Bright, and the non-bright white is grey.

Probably fine to keep them as-is so it won’t break anything.

I’ll try to get a doc change up over the weekend.

**Comment by joshorr** [comment]
Sorry, have not gotten to this yet.  I'll try to do it by end of week.

**Comment by borntyping** [comment]
No rush - this library's been around a while, fixes like this aren't urgent :)
